### PR TITLE
Add descriptions for upcoming reid models, remove old models

### PR DESCRIPTION
**Pull request by sovrasov**
@DmitriySidnev
@snosov1

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
why expected color order is RGB? Most of pre-trained models in OMZ expect BGR color order.

**Comment by DmitriySidnev** [comment]
Because these models were trained in RGB format. We used a trainig code that has Pillow for processing images by default. And pretrained weights were in RGB too.

**Comment by IRDonch** [comment]
> Because these models were trained in RGB format. We used a trainig code that has Pillow for processing images by default. And pretrained weights were in RGB too.

None of this means that the final model has to have RGB inputs. You can use the `--reverse_input_channels` MO flag to make them take BGR inputs. And you _should_, because the existing demos all expect ReID models to have BGR inputs (not to mention that BGR is the convention for all other Intel models).

**Comment by DmitriySidnev** [comment]
@IRDonch, could you please clarify who and where, in which config,  should use parameter `--reverse_input_channels`?

**Comment by eaidova** [comment]
@DmitriySidnev, this option performed automatically in internal model zoo for all models where specified RGB as input.

**Comment by DmitriySidnev** [comment]
@IRDonch , @vladimir-dudnik, please take a look. In description files was changed channels order for the new models to `BGR`. According to @eaidova comment we already have `RGB` order in input configs for the models and if MO automatically reverse channels it is right that here should be `BGR`.

File: models/intel/person-reidentification-retail-0103/description/person-reidentification-retail-0103.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+| Metric                            | Value                                     |
+|-----------------------------------|-------------------------------------------|
+| GlobalMe-reID rank@1 accuracy     | 93.5                                      |
+| GlobalMe-reID mAP                 | 69.5                                      |
+| Pose coverage                     | Standing upright, parallel to image plane |
+| Support of occluded pedestrians   | YES                                       |
+| Occlusion coverage                | <50%                                      |
+| GFlops                            | 0.564                                     |
+| MParams                           | 0.597                                     |
+| Source framework                  | Pytorch*                                  |
```
Comment (IRDonch) [inline models/intel/person-reidentification-retail-0103/description/person-reidentification-retail-0103.md]:
```suggestion
| Source framework                  | PyTorch*                                  |
```

File: models/intel/person-reidentification-retail-0103/description/person-reidentification-retail-0103.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+| Metric                            | Value                                     |
+|-----------------------------------|-------------------------------------------|
+| GlobalMe-reID rank@1 accuracy     | 93.5                                      |
+| GlobalMe-reID mAP                 | 69.5                                      |
+| Pose coverage                     | Standing upright, parallel to image plane |
+| Support of occluded pedestrians   | YES                                       |
+| Occlusion coverage                | <50%                                      |
+| GFlops                            | 0.564                                     |
+| MParams                           | 0.597                                     |
+| Source framework                  | PyTorch*                                  |
```
Comment (eizamaliev) [inline models/intel/person-reidentification-retail-0103/description/person-reidentification-retail-0103.md]:
```suggestion
| Source framework                  | PyTorch\*                                  |
```

File: models/intel/person-reidentification-retail-0103/description/person-reidentification-retail-0103.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+to locate at least one true positive in the top-1 rank.
+Mean Average Precision (mAP) is the mean across all queries' Average Precision (AP)
+and AP is defined as an area under the
+[precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall) curve.
+
+## Performance
+
+## Inputs
+
+1. name: "data" , shape: [1x3x256x128] - An input image in the format [BxCxHxW],
```
Comment (eizamaliev) [inline models/intel/person-reidentification-retail-0103/description/person-reidentification-retail-0103.md]: Remove number, it is not a list

Comment (sovrasov) [reply models/intel/person-reidentification-retail-0103/description/person-reidentification-retail-0103.md]: Fixed

**Comment by vladimir-dudnik** [comment]
@eizamaliev please review changes requested by you

File: models/intel/person-reidentification-retail-0103/description/person-reidentification-retail-0103.md (hunk 3 above)
Comment (eizamaliev) [reply models/intel/person-reidentification-retail-0103/description/person-reidentification-retail-0103.md]: For `Outputs` yes, but not for `Inputs`

Comment (sovrasov) [reply models/intel/person-reidentification-retail-0103/description/person-reidentification-retail-0103.md]: Now it's finally fixed

**Pull request merged**